mws/mws.py

```diff
@@ -1121,6 +1121,46 @@ def update_inbound_shipment(self, shipment_id, shipment_name,
             ))
         return self.make_request(data, method="POST")
 
+    def get_inbound_guidance_for_sku(self, skus=None, marketplace_id=None):
+        """
+        Returns inbound guidance for a list of items by Seller SKU
+        to help you decide if an item should be sent to a given marketplace.
+        """
+        marketplace_id = marketplace_id or 'ATVPDKIKX0DER'
+        skus = skus or []
+
+        # 'skus' should be a unique list, or there may be an error returned.
+        skus = utils.unique_list_order_preserved(skus)
+
+        data = dict(
+            Action='GetInboundGuidanceForSKU',
+            MarketplaceId=marketplace_id,
+        )
+        data.update(utils.enumerate_params({
+            'SellerSKUList.ID.': skus,
+        }))
+        return self.make_request(data, method="POST")
```

Comment:
I'm interested in `method="POST"`, the version in `develop` branch uses GET and it looks like this is a GET HTTP method. Does using `method="POST"` work for you?

Reply:
Using method="POST" works for me.

Reply:
The Amazon docs make no mention of this being a POST request either.

Reply:
@jkim65537 could you try the method as a GET to see if it works please?